web/themes/custom/pineapple/sass/layouts/nodes/_recipe.scss

```diff
@@ -99,3 +96,97 @@
     }
   }
 }
+
+.recipe--teaser {
+  display: flex;
+  flex-direction: column;
+  background-color: #f1f0f4;
+  width: 100%;
+  height: 100%;
+  overflow: hidden;
+
+  .recipe-card-wrapper {
+
+    .teaser-image-container {
+      position: relative;
+
+      .highlighted-label {
+        background-color: transparentize($primary, 0.4);
+        color: $white;
+        text-transform: uppercase;
+        text-align: left;
+        float: right;
+        position: absolute;
+        right: 0;
+        top: $unit * 3;
+        z-index: 1;
+        padding: 0 $unit;
+      }
+
+      img {
+        width: 100%;
+        object-fit: cover;
+      }
+    }
+
+    .node__content {
+      width: 100%;
+      padding: $unit * 3;
+
+      .recipe-labels {
+        display: flex;
+        align-content: center;
+        text-transform: uppercase;
+        color: $primary;
+        justify-content: space-between;
+
+        .field--name-field-r-cooking-time {
+          background-color: $white;
+          font-size: $xsmall-font;
+          text-align: center;
+          margin: $unit;
+          margin-left: 0;
+          padding: $unit $unit * 2;
+
+          .field__item {
+            display: block;
+          }
+        }
+        .field--name-field-r-serves {
+          display: flex;
+          align-items: center;
+          background-color: $white;
+          font-size: $xsmall-font;
+          text-align: center;
+          margin: $unit;
+          margin-right: 0;
+          padding: $unit / 2 $unit * 2;
```

Comment:
Remove margin and padding and implement padding as discussed

Reply:
done